main.go

```diff
@@ -77,6 +78,7 @@ func main() {
 		"Watch for custom resources in all namespaces, if set to false it will only watch the runtime namespace.")
 	flag.StringVar(&storagePath, "storage-path", "/data", "Where to store the persistent database of image metadata")
 	flag.Int64Var(&storageValueLogFileSize, "storage-value-log-file-size", 1<<28, "Set the database's memory mapped value log file size in bytes. Effective memory usage is about two times this size.")
+	flag.IntVar(&concurrent, "concurrent", 4, "The number of concurrent resource reconciles.")
```

Comment:
I think this flag definition comes from elsewhere; but either way, it deserves a better description, on two counts:

 - it's the _maximum_ number of workers
 - "reconciles" doesn't tell the user what it affects.

Something like "The maximum number of scans that can run concurrently" would be an improvement, I reckon. WDYT?

Reply:
Oh it was merged already. OK never mind.

Reply:
@squaremo I agree. I'll come up with a follow-up PR.

Reply:
We could extract this into `fluxcd/pkg/runtime` like we did with the log level and other common flags. The description used here is the same for all the others controllers, better to change it on place.